patches/minecraft/net/minecraft/command/arguments/ArgumentTypes.java.patch

```diff
@@ -0,0 +1,12 @@
+--- a/net/minecraft/command/arguments/ArgumentTypes.java
++++ b/net/minecraft/command/arguments/ArgumentTypes.java
+@@ -220,4 +220,9 @@
+          this.field_197481_c = p_i48088_3_;
+       }
+    }
++   @javax.annotation.Nullable public static ResourceLocation getId(ArgumentType<?> type) {
++      Entry<?> entry = func_201040_a(type);
++      return entry == null ? null : entry.field_197481_c;
++   }
++
```

Comment:
Another empty line.....
Sorry for being annoying about this. But smaller patches benefit everybody.

Reply:
All good, I am just being an idiot.